### PR TITLE
Gallery branch

### PR DESCRIPTION
**Pull request by Exgene**
## Features
- Updated gallery page for 2024
- Space gamey themed
- Game styles components
- Preview component for better viewing
- Custom loading bar

## Breaking Changes
- Update the Cloudinary URL
- Update the year array accordingly in `page/gallery/index.tsx`
- Look for potential bugs
- Custom keyframes and animation added in `tailwind.config.ts`
- Tooltip to be added

### CODE REVIEW

**Review by Aniruddha-Upadhya-K — APPROVED** [review]

**Review by swasthikshetty10 — APPROVED** [review]

**Review by nagarajpandith — APPROVED** [review]

File: src/components/galleryslide/devices/final-component.tsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    <div
+      className="w-full text-white flex flex-col items-center font-VikingHell"
+      id="animation"
+    >
+      <h1 className="font-bold text-4xl md:text-6xl tracking-widest">Incridea 24`</h1>;
+      <p className="text-sm md:text-lg tracking-wider">
+        Its your time to create great memories
+      </p>
+      <Image
+        src={"/assets/png/4.png"}
```
Comment (nagarajpandith) [inline src/components/galleryslide/devices/final-component.tsx:15]: should be
```       
src={"/assets/png/dice.png"}
```

File: src/components/galleryslide/progressBar/progress-bar.tsx [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+ }, [filledSteps, loadedSteps, year]);
+
+
+  return (
+    <div className={styles["progress-bar"]}>
+      {[...Array(totalYears)].map((_, index) => (
+        <div key={index} className={styles["progress-step"]}>
+          {index < filledSteps && (
+            <Image
+              src={`/assets/png/${index + 1}.png`}
```
Comment (nagarajpandith) [inline src/components/galleryslide/progressBar/progress-bar.tsx:75]: should be
```
src={`/assets/png/level${index === 0 ? "" : index}.png`}
```

**Review by nagarajpandith — CHANGES_REQUESTED** [review]

**Comment by nagarajpandith** [comment]
Wanted to confirm whether the Blue progressbar hidden behind navbar is intentional? @Exgene 
It is from swiper's pagination progressbar

Screenshot 2024-02-01 at 10 14 04 AM

**Comment by Exgene** [comment]
> Wanted to confirm whether the Blue progressbar hidden behind navbar is intentional? @Exgene It is from swiper's pagination progressbar
> 
> Screenshot 2024-02-01 at 10 14 04 AM

Yes i think its fine, Looks nice for the backdrop blur of the navbar, Can change color to match the main color scheme!

File: src/components/galleryslide/progressBar/progress-bar.tsx (hunk 2 above)
Comment (Exgene) [reply src/components/galleryslide/progressBar/progress-bar.tsx:75]: Oh no my assets are missing from here, Logic was fine. Will commit with a fix

**Review by nagarajpandith — APPROVED** [review]

**Pull request merged**